toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -415,10 +439,11 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 	}
 
 	// Calculate how many packages need to be installed so an accurate percent complete can be reported
-	totalPackages, err := calculateTotalPackages(packagesToInstall, installRoot)
+	installedPackages, err := calculateTotalPackages(packagesToInstall, installRoot)
```

Comment:
nit: can installedPackages be renamed to packagesToInstall? Sounds more accurate to its purpose

Reply:
packagesToInstall is already used for the input parameter to this function. The function returns a different list, which includes the dependencies of those in "packagesToInstall".

Reply:
How about `allInstalledPackages`, `packagesToInstallFullList`, etc.?